test/testModel.cc

```diff
@@ -1357,16 +1377,18 @@ SUITE(GodleyTableWindow)
       CHECK_EQUAL(1,insertIdx);
       keyPress(XK_Left,"");
       CHECK_EQUAL(0,insertIdx);
-      keyPress(XK_Left,"");
-      CHECK_EQUAL(0,selectedCol);
```

Comment:
We should check that the (1,0) cell isn't selected. What should happen - do we wrap (go to the bottom right cell) or do we do nothing? Either option are viable choices, I don't quite remember what was done.

Reply:
I now check that cell (1,0) is not selected by checking that selectedRow == SelectedCol == -1 (i.e., deselection). So, essentially nothing is done. If a button is now pressed, selectedRow and selectedCol will be set to the values on lines 632-643 of godleyTableWindow.cc. The latter has also been modified slightly on line 639: selectedRow = 2 when the down arrow is pressed from a deselected state, so that the first modifiable cell in column 0 becomes highlighted (no longer cell(1,0)).